cluster/ubuntu/util.sh

```diff
@@ -450,7 +451,8 @@ function provision-node() {
       '${KUBELET_CONFIG}'
     create-kube-proxy-opts \
       '${1#*@}' \
-      '${MASTER_IP}' 
+      '${MASTER_IP}' \
+      '${KUBE_PROXY_EXTRA_OPTS}'
```

Comment:
Should this be quoted or not? If it's quoted, can you still pass multiple options in the envvar?